sdk/cosmos/azure-cosmos-spark_3-0_2-12/Dev-README.md

```diff
@@ -0,0 +1,35 @@
+# Dev Azure Cosmos DB OLTP Spark connector client library for Java
+
+### How to run unit
+
+To run the tests of Spark connector without running the SDK tests you need to install azure-cosmos first
+```bash
+mvn -e -DskipTests  -Dgpg.skip -Dmaven.javadoc.skip=true -Dspotbugs.skip=true -Dcheckstyle.skip=true -Drevapi.skip=true -pl ,azure-cosmos -am clean install
+```
+
+To run unit tests:
+```
+mvn -e -Dgpg.skip -Dmaven.javadoc.skip=true -Dspotbugs.skip=true -Dcheckstyle.skip=true -Drevapi.skip=true -pl ,azure-cosmos-spark_3-0_2-12 test package -Punit
+```
+
+To run integration tests (requires Cosmos DB endpoint)
+
+Create the file ~/cosmos-v4.properties with the following content (modify to match your cosmos endpoint):
+
+```
+ACCOUNT_HOST=https://192.168.1.51:8081
```

Comment:
IMO we should start with the deafult localhost here - https://localhost:8081

Reply:
good catch. changed to localhost. thanks.